src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsJWPlayerObject.java

```diff
@@ -124,6 +124,10 @@ public void allowToPlayVideoForSomeTime(Duration duration) {
   }
 
   public void clickOnPlayer() {
-    playButton.click();
+    builder.moveToElement(playButton)
```

Comment:
It's funny current code looks much less stable than previous one. Yet, it works :( Can't we really make selenium to wait and click exactly the play button on a movie? One thing we could try use is to re-use [SmartBanner component](https://github.com/Wikia/selenium-tests/blob/b6b1489d9c9d715521a1f5eb11cd309a6c10ab08/src/test/java/com/wikia/webdriver/elements/communities/mobile/components/SmartBanner.java) and close it with its method. In terms of global nave I found its test use JS to scroll a hundred pixels: https://github.com/Wikia/selenium-tests/blob/eea8e8bb90c0a84547a7068340eefcb350adfbdc/src/test/java/com/wikia/webdriver/testcases/mobile/navigation/globalnav/GlobalNavigationMobileWikiTests.java#L35 I'm not sure it'll help us, though :(

Reply:
I've tried to use both. The other solution is to remove global nav from the dom, but I don't recommend this solution